tutorials/movies/customizing_movie_interface.md

```diff
@@ -262,9 +268,11 @@ InstanceName corresponds to singular name and is used in New Record (New Movie)
 namespace MovieTutorial.MovieDB.Entities
 {
     // ...
-    [ConnectionKey("Default"), DisplayName("Movies"), InstanceName("Movie"), 
-     TwoLevelCached]
-    public sealed class MovieRow : Row, IIdRow, INameRow
+    [ConnectionKey("Default"), Module("MovieDB"), TableName("[mov].[Movie]")]
+    [DisplayName("Movies"), InstanceName("Movie")]
+    [ReadPermission("Administration:General")]
```

Comment:
not sure why we use Administration:General for movie permissions here?

Reply:
[Generating Code For Movie Table doc page](https://github.com/serenity-is/docs/blob/master/tutorials/movies/generating_code_for_movie_table.md) here it says to leave permission as *Administration:General*